docs/source/docs/command-reference/context_free/merge.md

```diff
@@ -100,37 +92,59 @@ Parameters:
 - `-h, --help` - Print the help message and exit.
 - `-- <extra format args>` - Additional arguments for the format writer
   (use `-- -h` for help). Must be specified after the main command arguments.
+- intersect merge policy options:
+  - `-iou`, `--iou-thresh` (number) - IoU matching threshold for spatial
+    annotations (both maximum inter-cluster and pairwise). Default is 0.25.
+  - `-oconf`, `--output-conf-thresh` (number) - Confidence threshold for output
+    annotations to be included in the resulting dataset. Default is 0.
+  - `--quorum` (number) - Minimum count of votes for a label or attribute
+    to be counted. Default is 0.
+  - `-g, --groups` (string) - A comma-separated list of label names in
+    annotation groups to check. The `?` postfix can be added to a label to
+    make it optional in the group (repeatable)
 
 Examples:
-
-Merge 4 (partially-)intersecting projects,
-- consider voting successful when there are no less than 3 same votes
-- consider shapes intersecting when IoU >= 0.6
-- check annotation groups to have `person`, `hand`, `head` and `foot`
-(`?` is used for optional parts)
-
-``` bash
-datum merge project1/ project2/ project3/ project4/ \
-  --quorum 3 \
-  -iou 0.6 \
-  --groups 'person,hand?,head,foot?'
-```
-
-Merge images and annotations from 2 datasets in COCO format:
-`datum merge dataset1/:image_dir dataset2/:coco dataset3/:coco`
-
-Check groups of the merged dataset for consistency:
+- Merge 4 (partially-)intersecting projects,
+  - consider voting successful when there are no less than 3 same votes
+  - consider shapes intersecting when IoU >= 0.6
+  - check annotation groups to have `person`, `hand`, `head` and `foot`
+  (`?` is used for optional parts)
+
+  ```console
+  datum merge <path/to/project1/> <path/to/project2/> <path/to/project3/> <path/toproject4/> \
```

Comment:
```suggestion
  datum merge <path/to/project1/> <path/to/project2/> <path/to/project3/> <path/to/project4/> \
```

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/969/commits/bcc7a4dc3cda80689f191d08d7fe71991d34b05c